### DISCUSSION TITLE
[DEV] Add Prettier

### DISCUSSION THREAD

**Discussion by anishkny** [ryanluker/vscode-coverage-gutters#387] [Ideas]
Kudos on using `eslint` for enforcing best practices during development.

How about adding [`prettier`](https://prettier.io/) for consistent style as well?

I had to disable my VSCode Prettier extension while contributing #385 

I can send out a PR if this seems like a good idea.

**Comment by ryanluker** [comment]
@anishkny more automatic code styling the better imo!
Which default prettier format would we use? (I didn't develop the extension with a specific style so the first usage of prettier will produce a lot of changes most likely hah...)

**Reply by anishkny** [reply]
I usually just follow the plain vanilla Prettier config. Literally:

```bash
echo {} > .prettierrc.json
```

This helps prevent opinion-based debates about style. See: https://prettier.io/docs/en/why-prettier.html

Yes, first usage will produce a lot of changes. Let me send out a draft PR to see the upshot, but please don't feel obligated to merge - lets just see what we see...

**Reply by ryanluker** [reply]
Yep, makes sense to me, the less bike shedding the better IMO!
